lib/fetch-mix-deps.nix

```diff
@@ -24,7 +23,9 @@ let
       '';
 
       buildPhase = ''
-        mix deps.get
+        mix deps.get ${
+          stdenvNoCC.lib.optionalString (mixEnv == "prod") "--only-prod"
```

Comment:
`mix help deps.get` do not mention `--only-prod`, so if this work, then it is non-public interface which we cannot use. On the other hand there is `--only` flag that should work independently of the environment which we should probably use there. Alternatively with additional argument that would conditional disable it.

Reply:
Thanks for checking, I guess it's been deprecated. It still works but as you say, better use the public thing.